--- src/xalanc/XMLSupport/XalanOtherEncodingWriter.hpp	2025-04-01 11:21:27
+++ src/xalanc/XMLSupport/XalanOtherEncodingWriter.hpp_fix	2025-04-01 12:41:46
@@ -320,14 +320,14 @@
                 {
                     XalanUnicodeChar    value = decodeUTF16SurrogatePair(ch, theChars[i+1],  getMemoryManager());
 
-                    if (this->m_isPresentable(value))
-                    {
-                        write(value);
-                    }
-                    else
-                    {
-                        this->writeNumberedEntityReference(value);
-                    }
+                   // if (this->m_isPresentable(value))
+                   // {
+                   //     write(value);
+                   // }
+                   // else
+                   // {
+                   //     this->writeNumberedEntityReference(value);
+                   // }
 
                     ++i;
                 }
